kornia/augmentation/augmentation.py

```diff
@@ -22,8 +22,8 @@
 from kornia.filters import box_blur, gaussian_blur2d, motion_blur
 from kornia.geometry import (
     affine,
-    bbox_generator,
-    bbox_to_mask,
+    bbox_generator_2d,
```

Comment:
@edgarriba , is it really a good approach? `bbox_generator` will be 2d as default and `bbox_generator3d` will be for 3d -- is it really what we want?

The second thing is that some functions have to be renamed (i.e. `infer_bbox_shape` instead of `infer_box_shape`).

Reply:
Discussed this on slack.